src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Deprecation.php

```diff
@@ -18,6 +18,15 @@
  */
 class Deprecation
 {
+    private const PATH_TYPE_VENDOR = 'path_type_vendor';
+    private const PATH_TYPE_SELF = 'path_type_internal';
+    private const PATH_TYPE_UNDETERMINED = 'path_type_undetermined';
```

Comment:
I could have used an exception instead of 3-value return, but it did not feel "exceptional" enough. This is going to happen whenever a deprecation comes from the cache.

Reply:
hello PHP 5.5, we cannot use these consts

Reply:
fixed in 0c9b3c0